CONTRIBUTING.md

```diff
@@ -1,109 +1,143 @@
+* _help wanted_ - issue where we need help
+* _type: bug_ - issue for an error/ failure of Alva
+* _type: feature_ - issue for feature request
+* _type: design_ - issue for a design todo
+* _type: documentation_ - issue for needed, missing or unclear documentation
+* _type: feedback_ - issue for feedback and task where we need to figure out if it's a bug or we want it as a feature (or something else)
+* _type: question_ - issue for open question (most of the time, they need to be answered by core team)
+* _status: in progress_ - issue is currently in progress
+* _status: has PR_ - issue has already an open pull request
+* _priority: low_ - issue which is a nice to have, like a backlog issue
+* _priority: mid_ - issue needed in the near future, but not super urgent
+* _priority: high_ - issue must be resolved as soon as possible
 
 ### Working on issues
+
 Issues should be provided as branches starting with "feat/", as pull requests.
 
+## Coding conventions & guidelines
+
+### TypeScript & TSLint
 
-## Style Guide / Coding conventions
-### Coding conventions & guidelines
-#### TypeScript & TSLint
 We use strict tsconfig and TSLint rules, as well as Prettier. When it comes to naming things, we follow the [TypeScript guidelines](https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines), and we do not abbreviate names (except for trivial names like HTML).
 
-#### We use .editorconfig
+### We use .editorconfig
+
 Editorconfig helps developers define and maintain consistent coding styles between different editors and IDEs.
 There are a lot of plugins for editors/IDEs that support .editorconfig.
 
 You'll find more information, extensions for editors and IDEs at the [editorconfig page](http://editorconfig.org/).
 
+## Commit message conventions
 
-### Commit message conventions
 For commit message we use the [angular commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#commits)
-#### commitizen & commitlint
+
+### commitizen & commitlint
+
 https://github.com/marionebl/commitlint
 https://github.com/commitizen/cz-cli
 
-
 ## Team
-* Alexander Peschel
-* Julius Walther
-* Lasse Küchler
-* Markus Ölhafen
-* Tilman Frick
-* Thomas Jacob
 
+* [Alex Peschel](http://github.com/Alexpeschel)
+* [Alexander Bokov](http://github.com/alxbok)
+* [Daniel Gooß](http://github.com/Dangoo)
+* [Felicitas Kugland](http://github.com/kotzendekrabbe)
+* [Frederik Reiss](http://github.com/frederikreiss)
+* [Gregor Adams](http://github.com/pixelass)
+* [Julian Cebulla](http://github.com/Jumace)
+* [Lasse Küchler](http://github.com/LKuechler)
+* [Luca Oelsner](http://github.com/lucoel)
+* [Mario Nebl](http://github.com/marionebl)
+* [Markus Ölhafen](http://github.com/markusoelhafen)
+* [Thomas Jacob](http://github.com/TheReincarnator)
+* [Tilman Frick](http://github.com/tilmx)
 
 ## Questions?
+
 Just drop us an [email](alva@sinnerschrader.com) or write a direct message on Twitter to [@meetalva](https://twitter.com/meetalva)
```

Comment:
Can we change the mail address to hey@meetalva.io?

Reply:
Sure.

Reply:
Done.